packages/shared/src/components/tooltips/LinkWithTooltip.tsx

```diff
@@ -18,7 +18,7 @@ export function LinkWithTooltip({
     () =>
       React.cloneElement(children, {
         ...children.props,
-        'aria-label': tooltip.content,
+        'aria-label': typeof tooltip.content === 'string' && tooltip.content,
```

Comment:
Otherwise it might set `aria-label` to false

```suggestion
        'aria-label': typeof tooltip.content === 'string' ? tooltip.content : undefined,
```